.github/workflows/ci.yml

```diff
@@ -43,23 +60,11 @@ jobs:
         uses: taiki-e/install-action@nextest
 
       - name: Setup just
-        uses: extractions/setup-just@v1
+        uses: extractions/setup-just@v2
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Run sccache-cache
-        uses: mozilla-actions/sccache-action@v0.0.3
-
-      - name: Cache Rust
-        uses: Swatinem/rust-cache@v2
-
-      - name: Check license headers
-        run: just check-license-headers
-
-      - name: Run fmt
-        run: just check-fmt
-
-      - name: Run clippy
+      - name: Clippy Check
```

Comment:
How are we not running docker if clippy does not pass? From the commit message it reads as if this commit tries to do it.

Reply:
Ah, it was like this then I changed it back. Let me fix the commit message.